plugins/devtools-backend/src/service/router.ts

```diff
@@ -35,30 +35,68 @@ import { createPermissionIntegrationRouter } from '@backstage/plugin-permission-
 import {
   DiscoveryService,
   HttpAuthService,
+  LifecycleService,
+  LoggerService,
   PermissionsService,
 } from '@backstage/backend-plugin-api';
+import { SignalService } from '@backstage/plugin-signals-node';
 
 /** @public */
 export interface RouterOptions {
   devToolsBackendApi?: DevToolsBackendApi;
-  logger: Logger;
+  logger: LoggerService;
   config: Config;
   permissions: PermissionsService;
   discovery: DiscoveryService;
   httpAuth?: HttpAuthService;
+  lifecycle?: LifecycleService;
+  signalService?: SignalService;
```

Comment:
Yeah, requires a new PR. It's a bit breaking change too to rename it as it may be used already as is. Should we also export the `SignalService` as deprecated and remove it after next release?

Reply:
Here you go https://github.com/backstage/backstage/pull/23481